layouts/partials/head.html

```diff
@@ -17,6 +17,7 @@
     <script type="text/javascript" src="/js/feedback.js?v=2.0.0"></script>
     {{ if .IsHome }}
         <script type="text/javascript" src="/js/api_codes.js?v=2.0.0"></script>
+        <script type="text/javascript" src="/js/find_out_more.js?v=2.0.0"></script>
```

Comment:
@anusharanganathan is this file missing from the PR?

Reply:
@richard-jones no, it already exists. It was previously only used by the plugin. Now it's also used by the main site.